client/lib/olark-store/actions.js

```diff
@@ -45,6 +45,13 @@ const olarkActions = {
 		} );
 	},
 
+	setExpanded( isOlarkExpanded ) {
+		dispatcher.handleServerAction( {
+			isOlarkExpanded,
+			type: ActionTypes.OLARK_EXPANDED
```

Comment:
This is _slightly_ confusing. Maybe `OLARK_SET_EXPANDED` would be a clearer indication that the action is doing something.